guacamole-ext/src/main/resources/org/apache/guacamole/protocols/rdp.json

```diff
@@ -98,6 +98,7 @@
                         "it-it-qwerty",
                         "ja-jp-qwerty",
                         "sv-se-qwerty",
+			"es-es-qwerty",
```

Comment:
Indentation is off vs. the rest of the file - looks like there are three tabs on this line instead of spaces.  I think most of the other files you added are correct, but make sure you're following Guacamole style guidelines as documented here:
http://guacamole.apache.org/guac-style/